core/config/locales/en.yml

```diff
@@ -1532,6 +1532,9 @@ en:
     stock_transfers: Stock Transfers
     stop: Stop
     store: Store
+    store:
```

Comment:
@PSzyszkaSpark this is a duplicate key, there cannot be 2 keys with the same identifier